services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -15462,6 +15462,10 @@ components:
           type: string
           format: uuid
           title: Uid
+        createdAt:
```

Comment:
MINOR: try to increase the web-server version when you change the API. It is not mandatory but it is a good practice that eventually we should automate as well...